src/Components/Header/header.test.tsx

```diff
@@ -7,13 +7,13 @@ jest.mock("react-router-dom", () => ({
 }));
 test("renders Header component correctly", () => {
   render(<Header />);
-  const headerComponent = screen.getByAltText("Man Eating Popcorn");
+  const headerComponent = screen.getByLabelText("logo");
```

Comment:
Was this an intentional change?

Reply:
No totally fine. Just wanted to check it wasn't an unexpected change from a merge artifact from something. I see no issue in changing it. :)